sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -212,78 +213,80 @@
    * @param <T>                  The Type of the return, use byte[] to skip serialization.
    * @return A Mono Plan of type T.
    */
-  <T> Mono<Response<T>> invokeService(InvokeServiceRequest invokeServiceRequest, TypeRef<T> type);
+  <T> Mono<Response<T>> invokeMethod(InvokeServiceRequest invokeServiceRequest, TypeRef<T> type);
 
   /**
    * Invokes a Binding operation.
    *
-   * @param name      The name of the biding to call.
+   * @param bindingName      The bindingName of the biding to call.
    * @param operation The operation to be performed by the binding request processor.
    * @param data      The data to be processed, use byte[] to skip serialization.
    * @return an empty Mono.
    */
-  Mono<Void> invokeBinding(String name, String operation, Object data);
+  Mono<Void> invokeBinding(String bindingName, String operation, Object data);
 
   /**
    * Invokes a Binding operation, skipping serialization.
    *
-   * @param name      The name of the biding to call.
+   * @param bindingName      The name of the biding to call.
    * @param operation The operation to be performed by the binding request processor.
    * @param data      The data to be processed, skipping serialization.
    * @param metadata  The metadata map.
    * @return a Mono plan of type byte[].
    */
-  Mono<byte[]> invokeBinding(String name, String operation, byte[] data, Map<String, String> metadata);
+  Mono<byte[]> invokeBinding(String bindingName, String operation, byte[] data, Map<String, String> metadata);
 
   /**
    * Invokes a Binding operation.
    *
-   * @param name      The name of the biding to call.
+   * @param bindingName      The bindingName of the biding to call.
```

Comment:
nitpick: `The name of the binding to call.` 
also all occurrences of this argument.